maps/southern_cross/items/encryptionkey_vr.dm

```diff
@@ -0,0 +1,8 @@
+/obj/item/device/encryptionkey/pilot
+	icon_state = "cypherkey"
+	channels = list("Explorer" = 1)
+
+/obj/item/device/encryptionkey/pathfinder
+	name = "pathfinder's encryption key"
+	icon_state = "medsci_cypherkey"
+	channels = list("Command" = 1, "Science" = 1, "Medical" = 1, "Explorer" = 1)
```

Comment:
This job is equivalent to QM in that it is a 'sub-department lead position'. QM does not get command channel. In fact they get a normal cargo headset. Pathfinder should get a normal explorer headset.

I'm *strongly* opposed to them having Medical and Command, and I've constantly stressed that I think you're trying to make them a command position, which is not what we want. Stop doing it. Give them a normal key.